Samples/DynamicDependencies/Basic/cs-console-unpackaged/DynamicDependenciesBasicCS/DynamicDependenciesBasicCS/DynamicDependenciesBasicCS.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <Platforms>AnyCPU;x64</Platforms>
```

Comment:
might want to keep platforms consistent across the other C# samples in this repo